nvdaHelper/localWin10/sconscript

```diff
@@ -44,7 +44,7 @@ localWin10Lib = env.SharedLibrary(
 # Therefore, we must include it.
 # VS 2017 keeps changing the path to reflect the latest major.minor.build version which we canot easily find out.
 # Therefore  Search these versioned directories from newest to oldest  to collect all the files we need.
-vcRedistDirs = glob.glob(os.path.join(progFilesX86, r"Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.1*\x86\Microsoft.VC141.CRT"))
+vcRedistDirs = glob.glob(os.path.join(progFilesX86, r"Microsoft Visual Studio\2017\*\VC\Redist\MSVC\14.1*\x86\Microsoft.VC141.CRT"))

```

Comment:
If two different  variants of Visual Studio 2017 are installed, it is unclear as to which version of the dlls it will found. We really need to some how get the real path from Scons when it set up the visual studio paths. Though having a quick look myself so far this seems impossible. Unless we hack the vs took code in scons to save off a path during set-up.

Reply:
@michaelDCurran you are right, versions could mismatch.
SCons does not keep track of MSVC installation path once it used it to setup the environment.
Still, we can ask it to retrieve it again.
Pushed a second commit, using `SCons.Tool.MSCommon.vc.find_vc_pdir` which itself relies on calling `vswhere.exe` for 14.1.
If two different editions of Visual Studio 2017 are installed, I am still unsure as to which is used by SCons, but it will be the exact same for the compiler and the CRT.
Would you prefer I squash those commits together?